mmv1/templates/terraform/examples/cloudrun_vpc_access_connector.tf.erb

```diff
@@ -0,0 +1,78 @@
+# [START vpc_serverless_connector_enable_api]
+resource "google_project_service" "vpcaccess_api" {
+  service  = "vpcaccess.googleapis.com"
+  provider = google-beta
+}
+# [END vpc_serverless_connector_enable_api]
+
+# [START vpc_serverless_connector]
+# VPC
+resource "google_compute_network" "default" {
+  name                    = "<%= ctx[:vars]['cloudrun_network'] %>"
+  provider                = google-beta
+  auto_create_subnetworks = false
+}
+
+# VPC access connector
+resource "google_vpc_access_connector" "<%= ctx[:primary_resource_id] %>" {
+  name          = "<%= ctx[:vars]['vpcconn'] %>"
+  provider      = google-beta
+  region        = "us-west1"
+  ip_cidr_range = "10.8.0.0/28"
+  network       = google_compute_network.default.name
+}
+
+# Cloud Router
+resource "google_compute_router" "<%= ctx[:vars]['router'] %>" {
```

Comment:
You'll want to avoid using `:vars` entries for the Terraform resource name, and use a raw string instead like `"router"`. This line ends up being:

```
resource "google_compute_router" "router%{random_suffix}" {
```

Where Terraform can't find `google_compute_router.router.name` because this is called `google_compute_router.routerrouter%{random_suffix}` instead of `google_compute_router.router`

```suggestion
resource "google_compute_router" "router" {
```